dev-lua/compat53/compat53-0.14.3.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+LUA_COMPAT=( lua5-{1..4} luajit )
+
+inherit edo lua toolchain-funcs
+
+MY_PN="lua-${PN/53/-5.3}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="Compatibility module providing Lua-5.3-style APIs for Lua 5.2 and 5.1"
+HOMEPAGE="https://github.com/lunarmodules/lua-compat-5.3/"
+SRC_URI="https://github.com/lunarmodules/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+S="${WORKDIR}/${MY_P}"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64"
+
+REQUIRED_USE="${LUA_REQUIRED_USE}"
+
+DEPEND="${LUA_DEPS}"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	mv lbitlib.c bitlib.c || die
```

Comment:
A brief comment explaining this, maybe?

Reply:
each module is a part of the compat.'module' and so is called by app with that name. Rockspec do that too.